front/src/views/serials/serials.view.js

```diff
@@ -0,0 +1,17 @@
+
+export default {
+  name: 'ff-serials',
```

Comment:
for views it could be just serials-view
ff prefix is for components - not for views